.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+# Summary
+
+<<if there's an issue>>Fixes #issueNumber
+
+*High-level description of what your changes are accomplishing*
+
+*Add screenshots if applicable. Images are nice :)*
+
+# To Test
+
+1. <<Step one>> Open the page `about`
+- [ ] <<What to expect?>> Verify it contains about information...
+- [ ] Checkbox Style list of things a QA person could verify, i.e.
```

Comment:
Is this something they use? the checkbox. Afaik they have their tools and notes for the test, and adding checkbox might confuse if more than one people test? 

Im fine with leaving them, just making sure they like that or is useful. 


Normally we use the checkbox for unfinished parts of the PR.